tests/testsuite.src/used_binaries.at

```diff
@@ -1002,3 +1002,54 @@ AT_CHECK([$COBC -fdiagnostics-plain-output -fdiagnostics-show-caret -Wno-others
 
 AT_CLEANUP
 
+AT_SETUP([run profiling])
+AT_KEYWORDS([cobc profiling])
+
+AT_DATA([prog.cob], [
+        IDENTIFICATION DIVISION.
+        PROGRAM-ID.    prog.
+        PROCEDURE      DIVISION.
+        1ST SECTION.
+        PARA-0001.
+            PERFORM PARA-0003.
+        PARA-0002.
+            CONTINUE.
+        PARA-0003.
+            GO TO 2ND.
+        PARA-0004.
+            CONTINUE.
+        2ND SECTION.
+        PARA-0005.
+            PERFORM PARA-0006.
+        PARA-0006.
+            CONTINUE.
+        PARA-0007.
+            STOP RUN.
+])
+
+AT_CAPTURE_FILE([prof.csv])
+
+AT_CHECK([$COMPILE -fprof -x prog.cob], [0], [], [prog.cob: in section '1ST':
+prog.cob: in paragraph 'PARA-0003':
+prog.cob:11: warning: GO TO SECTION '2ND'
+])
+AT_CHECK([./prog], [0], [], [])
+
+#note: The time here is actually the number of times the procedure has been run, to avoid
```

Comment:
I _guess_ the idea was to base this on the testmode environment flag, but there's no such logic ion the code yet [it outputs the time]

Reply:
(cf. `cobprof.c:70`)